man/systemd.network.xml

```diff
@@ -997,6 +997,33 @@
         </varlistentry>
 
         <varlistentry>
+          <term><varname>RequestDHCPOptions=</varname></term>
+          <listitem>
+            <para>A whitespace-separated list of DHCP Parameter Request list option.
+            Defaults to none.</para>
```

Comment:
The documentation is a bit unclear to me. Does this option mean networkd will request subnet mask, routers, hostname, etc from the dhcp server and then apply it locally, or does it mean that networkd will send the currently configured subnet mask, routers, hostname, etc to the server?

The description of each option suggest one should pass a parameter but these are in fact flag options.